netket/models/equivariant.py

```diff
@@ -619,6 +636,8 @@ def GCNN(
             `lecun_normal(in_axis=1, out_axis=0)` which guarantees the correct variance of the
             output.
         bias_init: Initializer for the biases of all layers.
+        ensure_cplx: The return type is always complex, avoiding the possibility of NaNs
+            from taking the log of negative reals.
```

Comment:
can you better phrase that? I think you could say

```
ensure_cplx: If True the network returns a complex value. This is necessary to handle groups that have characters with a negative sign.
```

Reply:
By the way, we could also check the characters by hand, when constructing the model... but ok. Maybe not for now.